frontend/src/components/InterviewPage.js

```diff
@@ -89,24 +89,22 @@ const InterviewPage = () => {
 
     return (
         <>
-            <div className="d-flex justify-content-center mt-5"><h1>{questionTitle}</h1></div>
-            <div className="container mt-5">
+            <div className='d-flex justify-content-center mt-5'><h1>{questionTitle}</h1></div>
+            <div className='container mt-5'>
                 <ReactMarkdown>{questionDescription}</ReactMarkdown>
             </div>
-            <div>
-                <CodeEditor socket={collabSocket} roomId={interviewId} />
-            </div>
+            <CodeEditor socket={collabSocket} room={interviewId} />
             <Chat socket={chatSocket} username={username} room={interviewId} />
-            <div className="d-flex justify-content-center mt-5">
-                <Button variant="danger" onClick={handleShowEndInterview}>End interview</Button>
-                <Modal className="deleteModal" show={showEndInterview} onHide={handleCloseEndInterview} keyboard={false} animation={false}>
+            <div className='d-flex justify-content-center mt-5'>
+                <Button variant='danger' onClick={handleShowEndInterview}>End interview</Button>
+                <Modal className='deleteModal' show={showEndInterview} onHide={handleCloseEndInterview} keyboard={false} animation={false}>
```

Comment:
Should revert all the changes from single quotes back to double quotes, not consistent with the rest of frontend